tests/test_adjoint_jacobian.py

```diff
@@ -645,7 +645,7 @@ def circuit_ansatz(params, wires):
     qml.Rot(params[6], params[7], params[8], wires=wires[0])
     # #     qml.Rot(params[8], params[8], params[9], wires=wires[1]).inv()
     qml.MultiRZ(params[11], wires=[wires[0], wires[1]])
-    # #     qml.PauliRot(params[12], "XXYZ", wires=[wires[0], wires[1], wires[2], wires[3]])
+    qml.PauliRot(params[12], "XXYZ", wires=[wires[0], wires[1], wires[2], wires[3]])
```

Comment:
Are we testing correctness, or just that it gets called?

Reply:
The latter I believe.